responses/providers.go

```diff
@@ -0,0 +1,34 @@
+package responses
+
+import (
+	"dcfs/db/dbo"
+	"github.com/google/uuid"
+)
+
+type ProviderDataResponse struct {
+	UUID uuid.UUID `json:"uuid"`
+	Type int       `json:"type"`
+	Name string    `json:"name"`
+	Logo string    `json:"logo"`
+}
```

Comment:
czy to są dokładnie takie same pola jak objekt Provider? Może tutaj nie potrzebujemy nowego obiektu - moglibyśmy dopisać tylko wstawki jsonowe do obiektu Provider i dopisać, że jest on embedded i jego zwrócić?

Reply:
Racja, słuszna uwaga. Już poprawiłem 👍